lightkurve/periodogram.py

```diff
@@ -2,64 +2,251 @@
 from astropy import units as u
 from astropy.stats import LombScargle
 from matplotlib import pyplot as plt
+from scipy.ndimage.filters import gaussian_filter
+from astropy.units import cds
+from astropy.table import Table
+import astropy
 
-__all__ = ['Periodogram']
+import logging
+from . import PACKAGEDIR, MPLSTYLE
 
 
+log = logging.getLogger(__name__)
+
+__all__ = ['Periodogram', 'estimate_mass', 'estimate_radius',
+            'estimate_mean_density', 'stellar_params', 'standardize_units']
+
+###Add uncertainties <- review these [astero]
+numax_s = 3090.0 # Huber et al 2013
+deltanu_s = 135.1
+teff_s = 5777.0
+
+class InputError(Exception):
+    """Raised if user inputs both frequency and period kwargs"""
+    pass
+
 class Periodogram(object):
+    ###Update this [houseekeping]
     """Represents a power spectrum.
-
     Attributes
     ----------
+    lc : `LightCurve` object
+        The LightCurve from which the Periodogram is computed. Holds all  of
+        the `LightCurve object's properties.
     frequencies : array-like
         List of frequencies.
-    powers : array-like
-        Power measurements.
+    periods : array-like
+        List of periods (1 / frequency)
+    power : array-like
+        The power-spectral-density of the Fourier timeseries.
```

Comment:
If we use `periods` and `frequencies` as plural, should we also use the plural `powers` here?  (With docstring `List of powers of ....`)

On a related note: is `amplitudes` better than `power`, or does amplitude have a different meaning?

Reply:
Re: plural/singular Im not sure what to do here--- I started off writing frequencies and periods but as Im going along I find myself typing 'frequency' and 'period' and having to correct myself quite a lot, which makes me think that that might be the natural thing to type or just me. Im leaning towards changing everything to singular.

Re: amplitudes/power, the units on the y-axis are technically 'power spectral density', but I will have to do a bit more reading before I can justify and resolve this comment.

Reply:
Update:

Re: Plural/singular, I have now changed 'frequencies' to 'frequency' and 'periods' to 'period' to match the nomenclature in lightkurve as a whole.

Re: Amplitudes/Power: while using a LombScargle periodogram with Power Spectral Density normalisation (which we are), referring to the y-axis as 'power' for short and 'Power Spectral Density' in full (as is done when plotting) is appropriate. We can revisit when we include other periodogram methods such as BLS.